.travis.yml

```diff
@@ -1,9 +1,28 @@
+### Project specific config ###
 language: objective-c
 
+env:
+  matrix:
+    - ATOM_CHANNEL=stable
+    - ATOM_CHANNEL=beta
+
+install:
+  - brew update && brew install swiftlint
```

Comment:
swiftlint is pre-installed on Travis. So we should use:
```yaml
before_install:
  - brew update
  - brew outdated swiftlint || brew upgrade swiftlint
```
https://docs.travis-ci.com/user/osx-ci-environment/#A-note-on-upgrading-packages